src/EventStore.Core.Tests/Services/ElectionsService/ClusterSettingsFactory.cs

```diff
@@ -36,7 +36,9 @@ public class ClusterSettingsFactory {
 				TimeSpan.FromSeconds(10),
 				TimeSpan.FromSeconds(10),
 				TimeSpan.FromSeconds(10),
-				TimeSpan.FromSeconds(10), true, Opts.MaxMemtableSizeDefault, Opts.HashCollisionReadLimitDefault, false,
+				TimeSpan.FromSeconds(10), 
+				TimeSpan.FromMinutes(30),
```

Comment:
```suggestion
				TimeSpan.FromSeconds(1800),
```
Given that the option is in seconds, let's use seconds everywhere when setting the option.